package.json

```diff
@@ -27,6 +27,7 @@
         "typescript": "~4.5.4"
     },
     "dependencies": {
+        "@aws-quickstart/eks-blueprints": "^0.1.4",
```

Comment:
this is already in eks-blueprints, why do we need this dependency?

Reply:
removed - I ran into some weird dependency issues when i was testing but it ended up just being an issue with the version of CDK I was using